modules/retail/pages/containerized-saltboot.adoc

```diff
@@ -0,0 +1,158 @@
+[[retail-install-setup-containerized]]
+= Set Up the {productname} {smr} Environment using containerized proxy
+
+To set up the {productname} {smr} environment, you will need to have already installed and configured:
+
+* {productname} {smr} Server 4.3 or newer
+* one or more {productname} {smr} containerized proxies
+* one or more {productname} build host
+
+This section covers how to configure your  environment using containerized {productname} Proxy for [systemitem]``Saltboot`` deployment.
+
+[NOTE]
+====
+Containerized {productname} {smr} environment does not use {productname} {smr} Branch Servers.
+====
+
+[IMPORTANT]
+====
+Containerized workflow requires POS images build using {productname} Server 4.3 or newer.
+Older images will not work.
+====
+
+[IMPORTANT]
+====
+Containerized workflow not longer implicitly configure DHCP for PXE booting.
+See xref:specialized-guides:salt/salt-formula-dhcpd.adoc[] how to use DHCP formula to configure DHCP server.
```

Comment:
We might mention the possible need of DNS formula too.
At least, I required it in my manual setup.